app/scenes/Collection.tsx

```diff
@@ -48,14 +48,14 @@ function CollectionScene() {
   const can = usePolicy(collection?.id || "");
 
   React.useEffect(() => {
-    if (collection) {
+    if (collection?.name) {
```

Comment:
Why the change?

Reply:
the useEffect wasn't running after the update.